egs/wsj/s5/steps/nnet3/chain2/process_egs.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+# Copyright   2019  Johns Hopkins University (Author: Daniel Povey).  Apache 2.0.
+# Copyright   2019  Idiap Research Institute (Author: Srikanth Madikeri).  Apache 2.0.
+#
+# This script takes nnet examples dumped by steps/chain/get_raw_egs.sh and
+# combines the chunks into groups by speaker (to the extent possible; it may
+# need to combine speakers in some cases), locally randomizes the result, and
+# dumps the resulting egs to disk.  Chunks of these will later be globally
+# randomized (at the scp level) by steps/chaina/randomize_egs.sh
+
+
+# Begin configuration section.
+cmd=run.pl
+chunks_per_group=4
```

Comment:
Hm.  I think for normal models this chunks_per_group should default to 1, and num_repeats should default to 1 too.  Grouping utterances by speaker was necessary for some models I was investigating at the time, but not now.

Reply:
I have fixed this in the latest commit.